src/Sql/Sql/Database Backup/Services/AzureSqlDatabaseBackupAdapter.cs

```diff
@@ -279,6 +280,100 @@ internal AzureSqlDeletedDatabaseBackupModel GetDeletedDatabaseBackup(string reso
             };
         }
 
+        /// <summary>
+        /// Update a Long Term Retention backup.
+        /// </summary>
+        /// <param name="locationName">The location name.</param>
+        /// <param name="serverName">The server name.</param>
+        /// <param name="databaseName">The database name.</param>
+        /// <param name="backupName">The backup name.</param>
+        /// <param name="resourceGroupName">The resource group name</param>
+        /// <param name="onlyLatestPerDatabase">Whether or not to only get the latest backup per database.</param>
+        /// <param name="databaseState">The state of databases to get backups for: All, Live, Deleted.</param>
+        internal AzureSqlDatabaseLongTermRetentionBackupCopyModel CopyDatabaseLongTermRetentionBackup(
+            AzureSqlDatabaseLongTermRetentionBackupCopyModel model)
+        {
+            Management.Sql.Models.LongTermRetentionBackupOperationResult response = Communicator.CopyDatabaseLongTermRetentionBackup(
+                model.SourceLocation,
+                model.SourceServerName,
+                model.SourceDatabaseName,
+                model.SourceBackupName,
+                model.SourceResourceGroupName,
+                new Management.Sql.Models.CopyLongTermRetentionBackupParameters()
+                {
+                    TargetServerFullyQualifiedDomainName = model.TargetServerFullyQualifiedDomainName,
+                    TargetDatabaseName = model.TargetDatabaseName,
+                    TargetServerResourceId = model.TargetServerResourceId,
+                    TargetSubscriptionId = model.TargetSubscriptionId,
+                    TargetResourceGroup = model.TargetResourceGroupName
+                });
+
+            Management.Sql.Models.LongTermRetentionBackup sourceBackup = Communicator.GetDatabaseLongTermRetentionBackup(
+                model.SourceLocation,
+                model.SourceServerName,
+                model.SourceDatabaseName,
+                model.SourceBackupName,
+                model.SourceResourceGroupName);
+
+            Dictionary<string, string> targetBackupResourceIdSegments = ParseLongTermRentionBackupResourceId(response.ToBackupResourceId);
+
+            string targetLocationName = targetBackupResourceIdSegments["locations"];
+            string targetServerName = targetBackupResourceIdSegments["longTermRetentionServers"];
+            string targetDatabaseName = targetBackupResourceIdSegments["longTermRetentionDatabases"];
+            string targetBackupName = targetBackupResourceIdSegments["longTermRetentionBackups"];
+            string targetResourceGroupName = targetBackupResourceIdSegments.ContainsKey("resourceGroups") ? targetBackupResourceIdSegments["resourceGroups"] : null;
+
+            Management.Sql.Models.LongTermRetentionBackup targetBackup = Communicator.GetDatabaseLongTermRetentionBackup(
```

Comment:
why are you calling Get (target) LtrBackup in CopyBackup call? I think this won't work for BF decommission

Reply:
Won't the target backup be in a non-commissioned cluster?

Reply:
however it won't work for cross-cloud

Reply:
right; will remove and re-record copy test